source/manual/get-started.html.md

```diff
@@ -29,12 +54,179 @@ We use a Docker environment - [govuk-docker][] - for local development.
 
 👉 [Get setup with govuk-docker][govuk-docker]
 
+[govuk-docker]: https://github.com/alphagov/govuk-docker
+
 ## 3. Get SSH access to integration
 
-👉 [Get SSH access to integration](/manual/get-ssh-access.html)
+### Get access
+
+Ask somebody with access to add your SSH username (`firstnamelastname`) to the [user monitoring system][user-reviewer].
+
+[user-reviewer]: https://github.com/alphagov/govuk-user-reviewer
+
+### Create a user to SSH into integration
+
+User accounts in our integration environments are managed in the [govuk-puppet][] repository.
+
+    mac$ mkdir ~/govuk
```

Comment:
We can remove the `mac$` prefixes now that we no longer need to distinguish between host commands and VM commands.